packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "3d4a8d7caec7cd0239dbc40e39c8ec4d965a9a67",
-    "ref_origin" : "dcos-mesos-1.0.3-rc1"
+    "ref": "c7b7ca12e9a5d5a2bed92994f534868bda5c1bc9",
+    "ref_origin" : "dcos-mesos-1.0.x-17ac024"
```

Comment:
Looks like you updated `dcos-mesos-1.0.x-17ac024` to be based on `4154f66`, just like `dcos-mesos-1.0.x-4154f66` but the `buildinfo.json` still points to the wrong branch. Please fix.